planner/core/testdata/plan_suite_out.json

```diff
@@ -6026,11 +6026,11 @@
           "  │ ├─IndexRangeScan(Build) 250.00 cop[tikv] table:ta, index:idx_ta_name(name) range:[\"chad999\",\"chad99:\"), keep order:false, stats:pseudo",
           "  │ └─TableRowIDScan(Probe) 250.00 cop[tikv] table:ta keep order:false, stats:pseudo",
           "  └─MaxOneRow(Probe) 1.00 root  ",
-          "    └─HashAgg 1.00 root  funcs:sum(Column#17)->Column#13",
-          "      └─IndexLookUp 1.00 root  ",
-          "        ├─IndexRangeScan(Build) 10.00 cop[tikv] table:tb, index:idx_tb_id(id) range: decided by [eq(test.ta.id, test.tb.id)], keep order:false, stats:pseudo",
-          "        └─HashAgg(Probe) 1.00 cop[tikv]  funcs:sum(test.tb.code)->Column#17",
-          "          └─TableRowIDScan 10.00 cop[tikv] table:tb keep order:false, stats:pseudo"
+          "    └─StreamAgg 1.00 root  funcs:sum(Column#21)->Column#13",
```

Comment:
Is this affected by paging?

Reply:
Yes, it seems that the optimizer consider the new plan is better with paging available.